src/megameklab/com/printing/reference/DrivingSkillRollMods.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * MegaMekLab - Copyright (C) 2020 - The MegaMek Team
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ */
+package megameklab.com.printing.reference;
+
+import megamek.common.Entity;
+import megamek.common.EntityMovementMode;
+import megameklab.com.printing.PrintEntity;
+import megameklab.com.printing.PrintRecordSheet;
+import org.apache.batik.util.SVGConstants;
+
+/**
+ * Table for mods to vehicle driving skill checks
+ */
+public class DrivingSkillRollMods extends ReferenceTable {
+    private final Entity entity;
+
+    public DrivingSkillRollMods(PrintRecordSheet sheet, Entity entity) {
+        super(sheet, 0.02, 0.08, 0.14, 0.8);
+        this.entity = entity;
+        setColumnAnchor(0, SVGConstants.SVG_START_VALUE);
+        setColumnAnchor(1, SVGConstants.SVG_START_VALUE);
+        setColumnAnchor(2, SVGConstants.SVG_START_VALUE);
+
+        addMods();
+    }
+
+    public DrivingSkillRollMods(PrintEntity sheet) {
+        this(sheet, sheet.getEntity());
+    }
+
+    private void addMods() {
+        addRow(bundle.getString("unitActions"), "", "", "");
+        if (entity.getMovementMode().equals(EntityMovementMode.HOVER)
+                || entity.getMovementMode().equals(EntityMovementMode.VTOL)
+                || entity.getMovementMode().equals(EntityMovementMode.WIGE)) {
+            addRow("", bundle.getString("flankingMovement"), "",
+                    bundle.getString("possibleSideslip"));
+            addRow("", bundle.getString("afterFacingChange"), "", "");
+            addRow(bundle.getString("sideslipMovement"), "", "", "");
+        } else {
+            addRow("", bundle.getString("flankingMovement"), "",
+                    bundle.getString("possibleSkid"));
+            addRow("", bundle.getString("afterFacingChange"), "", "");
+            addRow("", bundle.getString("onPavement"), "", "");
+            addRow(bundle.getString("skiddingMovement"), "", "", "");
+        }
+        addRow("", bundle.getString("hexesMovedInTurn"), "", "");
+        addRow("", "", "0-2", "-1");
```

Comment:
This seems like it's off by 1, is this not "to-hit" modifiers for when the unit is being shot at?

Reply:
This is the driving skill check to avoid skidding when moving at flanking speed on pavement after a facing change.